learn/smartcontracts/README.md

```diff
@@ -3,24 +3,26 @@
 In this section, we explain what makes smart contracts on Tezos unique, and go over the powerful high-level languages you can choose from to write those smart contracts.
 
 1. [Michelson](michelson.md)
-2. [SmartPy](smartpy.md)
-3. [LIGO](ligo.md)
-4. [Morley/Lorentz ](untitled.md)
-5. [Archetype](archetype.md)
-6. [Formal Verification for Smart Contracts](michelsonandcoq.md)
+1. [SmartPy](smartpy.md)
+1. [LIGO](ligo.md)
+1. [Morley](morley.md)
+1. [Lorentz](lorentz.md)
+1. [Indigo](indigo.md)
+1. [Cleveland](cleveland.md)
+1. [Archetype](archetype.md)
+1. [Formal Verification for Smart Contracts](michelsonandcoq.md)
```

Comment:
Oh, this is actually a nice improvement, keeping the enumeration correct manually isn't the good way to go.